package.json

```diff
@@ -6,6 +6,7 @@
   "main": "server/index.js",
   "scripts": {
     "build": "webpack",
+    "watch": "webpack --watch --mode=development --devtool=source-map",
```

Comment:
Your `dev` script does this for you, you just need to go to `http://localhost:3000` when it's running.